CrazyCanvas/Source/Match/MatchServer.cpp

```diff
@@ -296,3 +298,39 @@ bool MatchServer::OnClientConnected(const LambdaEngine::ClientConnectedEvent& ev
 
 	return true;
 }
+
+bool MatchServer::OnFlagDelivered(const OnFlagDeliveredEvent& event)
+{
+	using namespace LambdaEngine;
+
+	if (m_pLevel != nullptr)
+	{
+		uint32 numFlags = 0;
+		Entity* pFlags = m_pLevel->GetEntities(ELevelObjectType::LEVEL_OBJECT_TYPE_FLAG, numFlags);
+
+		if (numFlags > 0)
+		{
+			Entity flagEntity = pFlags[0];
+			FlagSystemBase::GetInstance()->OnFlagDropped(flagEntity, glm::vec3(0.0f, 2.0f, 0.0f));
+		}
+	}
+
+	uint32 newScore = GetScore(event.TeamIndex) + 1;
+	SetScore(event.TeamIndex, newScore);
+
+	const ClientMap& clients = ServerSystem::GetInstance().GetServer()->GetClients();
+
+	if (!clients.empty())
+	{
+		ClientRemoteBase* pClient = clients.begin()->second;
+
+		NetworkSegment* pPacket = pClient->GetFreePacket(PacketType::TEAM_SCORED);
+		BinaryEncoder encoder(pPacket);
+		encoder.WriteUInt32(event.TeamIndex);
+		encoder.WriteUInt32(newScore);
+
+		pClient->SendReliableBroadcast(pPacket);
+	}
+
+	return true;
+}
```

Comment:
Why does this function have bool as a return value when it always returns true? is it due to consistency?